### ISSUE TITLE
A11y test and remove alpha from MediaPlayer

### ISSUE THREAD

**Issue by pjlee11** [BBC-archive/psammead#2278]
**Is your feature request related to a problem? Please describe.**
The Media Player component is currently an alpha component and needs some final alpha checks. 

**Describe the solution you'd like**
Go through the a11y checklist ensuring the MediaPlayer performs as expected. 

**Describe alternatives you've considered**
N/A

**Testing notes**
[Tester to complete]

Dev insight: Will there be any potential regression? etc

- [x] This feature is expected to need manual testing.

**Additional context**
We have already highlighted a11y bugs that are outside of out control - https://docs.google.com/spreadsheets/d/1i0JonoJCEVGpsSQoS0SJj9nolT1RQOPxC2qjq9Xnc4o/edit#gid=0

**Comment by pjlee11** [comment]
**A11y iframe best practice** 

1) add a title attribute to the iframe EG: `<iframe title="some title">`
  [source](https://webaim.org/techniques/frames/#iframe)
Confirmed with @greenc05 that UX are looking into the use of a `title` attribute to improve the screen reader experience, a previous PR for this is [here](https://github.com/bbc/psammead/pull/2283)

~2) scrolling="no" should not be set 
  [source](https://webaim.org/techniques/frames/#iframe)
If the iframe wrapper becomes smaller than iframe contents the user should be able to scroll, see GIF below from Firefox~
![Scolling an iframe](https://user-images.githubusercontent.com/7791726/65967679-efecb300-e459-11e9-85a7-37d4c6508b9d.gif)
**Completed in #2280** 

~3) the iframe should have a <title> 
  [source](https://www.w3.org/TR/WCAG20-TECHS/H64.html) -> [2.4.2 Page Titled: Web pages have titles that describe topic or purpose. (Level A)](https://www.w3.org/TR/2008/REC-WCAG20-20081211/#navigation-mechanisms-skip)~
**Confirmed with @greenc05 this is not required**

~4) the iframe HTML must have lang attribute
    Source: bbc-a11y - In frame //div[@id='root']/main/div/div/div/div/iframe : html tag has no lang attribute: /html~
**Completed by https://github.com/bbc/simorgh/pull/4112**

~5) the iframe should have exactly one main heading
    Source: bbc-a11y - In frame //div[@id='root']/main/div/div/div/div/iframe : Found 0 h1 elements.~

~6) the iframe should have exactly one main landmark
    Source: bbc-a11y - In frame //div[@id='root']/main/div/div/div/div/iframe : Found 0 elements with role="main".~

Confirmed with @greenc05 that 5 and 6 are bugs in `bbc-a11y` so are not required.

**Comment by pjlee11** [comment]
1 and 2 can be done in the Psammead component. 3, 4, 5 and 6 will need to be done in Mozart + Morph

**Comment by 12** [comment]
@pjlee11 Regarding the second point, unless I am mistaken, we shouldn't ever see a scroll on the iframe element, as it should fill 100% of its parent container?

https://codepen.io/rhellyar/pen/gOYVxWO

So removing `scrolling="no"` is fine, but we should still not see a scrolling iframe.

**Comment by pjlee11** [comment]
@12 that is correct for AV using the classic skin. On the audio skin the height is statically set so if you zoom in the screen it is possible to have some of the content in the iframe cut off, which in this case we would expect to see scrolling in the iframe (massive edge case though)

**Comment by pjlee11** [comment]
This [alt text value on the canonical Media Player placeholder](https://github.com/bbc/psammead/blob/0eac2370cf1200bcdfdddc70f1a11f2c9b2b58c3/packages/components/psammead-media-player/src/Placeholder/index.jsx#L17) needs to be passed in as a prop for the component to be a11y compliant

Update: This should be completed by https://github.com/bbc/psammead/issues/1840

**Comment by greenc05** [comment]
@pjlee11 -  Can we pass an empty value for the alt attribute please?

**Comment by pjlee11** [comment]
@greenc05 Is that valid for a11y though? The canonical AV solution for articles is that we render an `<image>` with a play button overlay, then only if the user clicks the `<image>` do we render in the SMP instance using an `<iframe>`. Therefore, I would have thought the image should be considered as a normal `<image>` HTML element but with click functionality ...

**Comment by aeroplanejane** [comment]
Relates to https://github.com/bbc/simorgh/issues/3961
@greenc05 @pjlee11 can we simply pass the h1 page title as the title attribute e.g. BBC 코리아 라디오 on this page: https://www.bbc.com/korean/bbc_korean_radio/liveradio - I imagine on AV pages the iframe attribute title would be the title of the AV clip

**Comment by pjlee11** [comment]
The alpha tag will be removed in https://github.com/bbc/simorgh/issues/3961#issuecomment-543098600
